views/index.ejs

```diff
@@ -57,7 +57,18 @@
       </form>
       <div class="search_form__result"></div>
     </div>
-    <script>window.times = {init : Date.now()}</script>
+    <script>
+      window.times = {init : Date.now()}
+    </script>
+    <% if (noUi) { %>
+    <script>
+      // iframe: hide UI immediately to avoid content flashing while the JS bundle loads
+      if (location.search.indexOf('&no_ui=1') != -1) {
```

Comment:
this `if` clause can be removed now that the check is done on server side.

Reply:
oops, I overlooked that, sorry, fixed!